perf/src/sigverify.rs

```diff
@@ -271,6 +277,37 @@ fn do_get_packet_offsets(
         return Err(PacketError::InvalidPubkeyLen);
     }
 
+    // read instruction length
+    let instructions_len_offset = blockhash_offset
+        .checked_add(size_of::<Hash>())
+        .ok_or(PacketError::InvalidLen)?;
+
+    // Packet should have at least 1 more byte for instructions.len
+    let _ = instructions_len_offset
+        .checked_add(1usize)
+        .filter(|v| *v <= packet.meta().size)
+        .ok_or(PacketError::InvalidLen)?;
+
+    let (instruction_len, instruction_len_size) = packet
+        .data(instructions_len_offset..)
+        .and_then(|bytes| decode_shortu16_len(bytes).ok())
+        .ok_or(PacketError::InvalidLen)?;
+
+    // SIMD-0160: skip if has more than 64 top instructions
+    if instruction_len > solana_transaction_context::MAX_INSTRUCTION_TRACE_LENGTH {
+        return Err(PacketError::InvalidProgramLen);
```

Comment:
nit: I'm not really sure this is a reasonable name for the error we're encountering here? It should be `InvalidNumberOfInstructions` or something. There's only one use-case (for checking is_simple_vote) and it's can be the same name there, it's an invalid number of instructions?

Propose we rename it to be clearer in both uses